desc/magnetic_fields/_core.py

```diff
@@ -359,7 +360,11 @@ def compute_Bnormal(
         chunk_size : int or None
             Size to split computation into chunks of evaluation points.
             If no chunking should be done or the chunk size is the full input
-            then supply ``None``. Default is ``None``.
+            then supply ``None``.
+        B_plasma_chunk_size : int or None
```

Comment:
It's not immediately clear how this is different from the other chunk size parameter. Might be clearer if when both are present, we make clear that `chunk_size` is for the biot-savart calculation